dbos/_sys_db.py

```diff
@@ -104,19 +106,12 @@ class GetWorkflowsInput:
     Structure for argument to `get_workflows` function.
 
     This specifies the search criteria for workflow retrieval by `get_workflows`.
-
```

Comment:
Can we bring back the comment block?

Reply:
It's still there!  Those comments were all duplicated